### PR TITLE
Add socket and grpc integration docs

### PR DESCRIPTION
**Pull request by hossein-raeisi**
documentation for changes in [this](https://github.com/getsentry/sentry-python/pull/1911) pull request on python sdk

### CODE REVIEW

File: src/platforms/python/common/configuration/integrations/grpc.mdx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+description: "Learn about the gRPC integration and how it adds support for the grpcio grpc client and server."
+sidebar_order: 80
+---
+
+The [gRPC](https://grpc.io/) integration instruments all incoming requests and outgoing unary-unary, unary-stream grpc
+requests using grpcio channels.
+
+Use this integration to start(or continue) transactions for incoming grpc requests create spans for outgoing requests
+and ensure traces are properly propagated to downstream services.
+
```
Comment (lizokm) [inline src/platforms/python/common/configuration/integrations/grpc.mdx:12]:
```suggestion
Use this integration to start or continue transactions for incoming grpc requests, create spans for outgoing requests,
and ensure traces are properly propagated to downstream services.

```

File: src/platforms/python/common/configuration/integrations/grpc.mdx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+Install `sentry-sdk` from PyPI with the `grpcio` extra.
+
+```bash
+pip install --upgrade 'sentry-sdk[grpcio]'
+```
+
+## Configure
+
+Add appropriate interceptor to your grpc stub/server
```
Comment (lizokm) [inline src/platforms/python/common/configuration/integrations/grpc.mdx:23]:
```suggestion
Add appropriate interceptor to your grpc stub/server:
```

File: src/platforms/python/common/configuration/integrations/socket.mdx [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+title: SOCKET
+description: "Learn about the Socket integration and how it adds support network actions."
+sidebar_order: 80
+---
+
+Use this integration to create spans for dns resolves and connection creations
```
Comment (lizokm) [inline src/platforms/python/common/configuration/integrations/socket.mdx:7]:
```suggestion
Use this integration to create spans for dns resolves and connection creations.
```

**Review by lizokm — APPROVED** [review]
Thanks for contributing! This looks good to me.

**Comment by antonpirker** [comment]
Sorry for not merging yet, but we wait until the gRPC and Socket Integrations have been merge and deployed in the Python SDK to merge this PR. 

This will happen probably next week

File: src/platforms/python/common/configuration/integrations/grpc.mdx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+server = grpc.server(
+    thread_pool=...,
+    interceptors=[ServerInterceptor()],
+)
+```
+
+Client
+```python
+from sentry_sdk.integrations.grpc.server import ClientInterceptor
```
Comment (antonpirker) [inline src/platforms/python/common/configuration/integrations/grpc.mdx:38]:
```suggestion
from sentry_sdk.integrations.grpc.client import ClientInterceptor
```

**Pull request merged**